docs/UserGuide.md

```diff
@@ -90,18 +90,38 @@ gradlew run -Dargs="-view output_path/reposense-report"
 ```
 `-Dargs="..."` uses the same argument format as mentioned above.
 
-## CSV Config File
-The `CSV Config File` contains the list of repositories, and the corresponding target authors to track contribution of.
+## CSV Config Files
+#### Repository configuration file
+to configure the list of repositories to analyze and the respective repository level options.
+
 [repo-config.csv](../repo-config.csv) is an example of a configuration file setup. It should contain the following columns:
 
+Column Name | Explanation
+----------- | -----------
+Repository's Location | The `GitHub URL` or `Disk Path` to the git repository
+Branch | The branch to analyse in the target repository
+[Optional] Ignore Global List | The list of file path globs to ignore during analysis for each author. More details on the Java glob standard [here](https://javapapers.com/java/glob-with-java-nio/)
+
+#### Author configuration file [Optional]
+to configure the list of authors to analyse and the options.
+
+[author-config.csv](../author-config.csv) is an example of a configuration file setup. It should contain the following columns:
+
+```
+Note:
+If author-config.csv is not used or repositories that are not in author-config.csv,
```

Comment:
not used or **the repositories being analyzed are not specified by any authors in author-config.csv**?